tests/app/questionnaire/test_placeholder_transforms.py

```diff
@@ -118,6 +118,25 @@ def test_concatenate_list(self):
             == "Milk, Eggs, Flour, Water"
         )
 
+    def test_add(self):
+        assert self.transforms.add(1, 2) == 3
+        assert self.transforms.add("1", 2) == 3
+
+    def test_format_ordinal_prefix(self):
+        assert self.transforms.format_ordinal(1, "a_or_an") == "a 1st"
+        assert self.transforms.format_ordinal(2, "a_or_an") == "a 2nd"
+        assert self.transforms.format_ordinal(3, "a_or_an") == "a 3rd"
+        assert self.transforms.format_ordinal(4, "a_or_an") == "a 4th"
+        assert self.transforms.format_ordinal(8, "a_or_an") == "an 8th"
+        assert self.transforms.format_ordinal(11, "a_or_an") == "an 11th"
+
+    def test_format_ordinal_suffix(self):
+        assert self.transforms.format_ordinal(1, None) == "1st"
+        assert self.transforms.format_ordinal(2, None) == "2nd"
+        assert self.transforms.format_ordinal(3, None) == "3rd"
+        assert self.transforms.format_ordinal(4, None) == "4th"
```

Comment:
Given the behaviour, I think there should be tests for:
- 11th, 12th and 13th
- 21st, 22nd, 23rd
- 111th, 112th and 113th

Reply:
Its 1, 2, 3, 4, 8, 11, 12, 13, 18, 21, 22, 23, 111, 112, 113 now. Was I supposed to keep first four in there?